src/Pages/SelectedCategory/SelectedCategory.jsx

```diff
@@ -1,47 +1,108 @@
-import { useEffect } from "react";
-import { useState } from "react";
-import axios from "axios";
+import { useEffect, useState } from "react";
 
 import "../../Pages/SelectedCategory/SelectedCategory.css";
-import { useParams } from "react-router-dom";
+import { useNavigate, useParams, Link } from "react-router-dom";
 import { useLoader } from "../../Custom-hook/use-loader";
-
+import { useData } from "../../Context/data-context";
+import { getVideos } from "../../Data/getvideos";
+import { Header } from "../../components/Header/Header";
+import { Sidebar } from "../../components/Sidebar/Sidebar";
+import { Modal } from "../../components/Modal/Modal";
+import { addToHistory } from "../../Utility/add-to-history";
+import { useHistory } from "../../Context/history-context";
+import { usePlaylist } from "../../Context/platlist-context";
+import { useWatchLater } from "../../Context/watch-leter-context";
+import { deleteFromWatchLater } from "../../Utility/deleteFromWatchLater";
+import { addToWatchLater } from "../../Utility/addToWatchLater";
+import { useAuth } from "../../Context/auth-context";
 export const SelectedCategory = () => {
```

Comment:
if you have more than two or three file into one folder make index.js file import all file her and export it here 

and import all this file from your index.js file 
so it's looks clean